packages/atomic/src/components/facets-v1/facet-value-box/facet-value-box.tsx

```diff
@@ -30,7 +31,7 @@ export const FacetValueBox: FunctionalComponent<FacetValueProps> = (props) => {
         </span>
         <span
           part="value-count"
-          class="text-neutral-dark with-parentheses w-full text-sm mt-1"
+          class="text-neutral-dark ellipsed with-parentheses w-full text-sm mt-1"
```

Comment:
Instead of ellipse-ing the value count, I think we should ensure that it's always part of the tooltip and hide it from the visual element when it overflows.

Reply:
What do you mean by `ellipsed`? Is it a way of hiding it?

Reply:
I'll add a title, but I'm against hiding the value itself, most users will think the count is buggy cause it's not there